src/py/flwr/simulation/run_simulation.py

```diff
@@ -297,20 +297,22 @@ def _run_simulation(
     """
     if backend_config is None:
         backend_config = {}
+        backend_config["init_args"] = {}
```

Comment:
I made it `init_args` based on our discussion in #3463. For me it feels more logical to name it `init_args` as there is already a `backend_name` argument as well.  Since this is the `backend_config`, it should then thus be related to the backend set in my opinion. Naming it `ray_init_args` suggests that there are backend settings to be set that are not related to ray even though ray is the selected backend (or any other backend set through `backend_name`. If this is the case I agree that it should be `ray_init_args` though.

Reply:
I 100% agree with the logic you present. Let's keep it as `init_args`.